pkg/R/RDD.R

```diff
@@ -1100,9 +1137,7 @@ setMethod("saveAsObjectFile",
             if (!rdd@env$serialized) {
               rdd <- reserialize(rdd)
             }
-            .jcall(getJRDD(rdd), "V", "saveAsObjectFile", path)
-            # NOTE: rJava doesn't check for exceptions if the return type is void
```

Comment:
We catch all exceptions on the Java side when the method executes and return an error when we get an exception. So this regression shouldn't happen.